piptools/warnings.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import
+
+import sys
+import warnings
+
+
+class PipToolsBaseWarning:
+    pass
+
+
+class PipToolsPendingDeprecationWarning(PipToolsBaseWarning, PendingDeprecationWarning):
+    """
+    Deprecates things in future versions.
+    """
+
+    pass
```

Comment:
This is redundant, a docstring is already a no-op node in the class.
```suggestion
```

Reply:
Good call! Thanks.

Reply:
Addressed in 5f8aa2f.